extraTs/pattern/diamond.ts

```diff
@@ -0,0 +1,25 @@
+// #39515 Diamond pattern
+const createDiamondShape = (size)=>{
+    for(var i=1;i<=size;i++){
+       for(var s=size-1;s>=i;s--){
+          process.stdout.write(" ");
+       }
+       for(var j=1;j<=i;j++){
+          process.stdout.write("* ")
+       }
+       console.log();
+    }
+    if(i==size+1){
+       for(var i=1;i<=size-1;i++){
+          for(var s=1;s<=i;s++){
+             process.stdout.write(" ");
+          }
+          for(j=i;j<=size-1;j++){
+             process.stdout.write("* ");
+          }
+          console.log();
+       }
+    }
+ }
+
+ createDiamondShape(5);
```

Comment:
Add input type and return type of function

Reply:
done